package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-fh-build",
   "description": "A plugin for development and build lifecycle of FeedHenry components",
-  "version": "0.3.4",
+  "version": "0.3.5",
```

Comment:
Since this is adding new functionality, we should bump the minor version rather than the patch version, to stick with semver.